auth-web/src/components/auth/manage-business/SearchBusinessNameRequest.vue

```diff
@@ -139,7 +139,7 @@ export default class SearchBusinessNameRequest extends Vue {
     addNRDialog: InstanceType<typeof ModalDialog>
     manageBusinessDialog: InstanceType<typeof ManageBusinessDialog>
   }
-  addBusiness = async (loginPayload: LoginPayload) => {
+  addBusinessToList = async (loginPayload: LoginPayload) => {
```

Comment:
Change method name to avoid the following error: 
![image](https://github.com/bcgov/sbc-auth/assets/122643780/790bde02-aca2-4015-bb0b-c1800f68ce42)